lib/ansible/modules/packaging/os/flatpak.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+DOCUMENTATION = '''
+---
+module: flatpak
+version_added: "2.4"
+requirements:
+    - flatpak
+author:
+    - "John Kwiatkoski (@jaykayy)"
+short_description: "Install and remove flatpaks."
+description:
+    - "The flatpak module allows users to manage installation and removal of flatpaks."
+options:
+  name:
+    description:
+      -
+        "When state is set to `present`, `name` is best used as `http(s)` url format.
```

Comment:
I believe that this section should start on the same line as the dash instead of the line beneath from looking at other modules.